tools/exploredb/main.go

```diff
@@ -339,9 +339,11 @@ func printStates(stateC <-chan *modifiedState, doneC chan<- bool) {
 		roots, err := st.HistoricalRoots()
 		if err != nil {
 			log.WithError(err).Error("could not get historical roots")
+		} else {
+			size, count = sizeAndCountOfByteList(roots)
+			log.Infof("historical_roots              : size = %s, count = %d", humanize.Bytes(size), count)
 		}
-		size, count = sizeAndCountOfByteList(roots)
-		log.Infof("historical_roots              : size = %s, count = %d", humanize.Bytes(size), count)
+
```

Comment:
nitpick: not a fan of this whitespace formatting. I've been leaning more towards using `WithField` to make structured logs easier.

Reply:
It's a little unrelated to this PR